app/js/trouble.js

```diff
@@ -53,15 +55,15 @@
 			$dropdown.data("status", 'closed');
 			$dropdown.removeClass('gbl_dropdown_active');
 			Array.prototype.foreach = function(cb) {
-				for (i = 0; i < this.length; ++i) {
+				for (var i = 0; i < this.length; ++i) {
 					cb(this[i]);
 				}
-			}
-			while (true) setTimeout(function() { setDropdownHeight(); }, 1000)
+			};
+			while (true) setTimeout(function() { setDropdownHeight(); }, 1000);

```

Comment:
The issue jshint is warning about here is "making" a function inside a loop. Because the anonymous function expression being passed to setTimeout is essentially declaring a function, what you really want to do is move this outside of the loop - either declare a function outside the loop or assign the anonymous function to a variable, then pass in either the function name or the variable name as the first argument to setTimeout.